sources/models/src/modeled_types/kubernetes.rs

```diff
@@ -460,11 +460,11 @@ mod test_kubernetes_bootstrap_token {
 
 // =^..^=   =^..^=   =^..^=   =^..^=   =^..^=   =^..^=   =^..^=   =^..^=   =^..^=
 
-/// KubernetesEvictionHardKey represents a string that contains a valid Kubernetes eviction hard key.
+/// KubernetesEvictionKey represents a string that contains a valid Kubernetes eviction hard key.
```

Comment:
Nit: should remove "hard" from the description as well?
```suggestion
/// KubernetesEvictionKey represents a string that contains a valid Kubernetes eviction key.
```